analyzer/util/util.go

```diff
@@ -64,6 +64,9 @@ func (b *Backoff) Success() {
 
 // Failure increases the backoff interval.
 func (b *Backoff) Failure() {
+	if b.currentTimeout == 0 {
+		b.currentTimeout = 10 * time.Millisecond
```

Comment:
:information_source: 10ms here instead of 100ms used before (albeit incorrectly)